action.yml

```diff
@@ -410,7 +398,7 @@ runs:
          if [ -d $HOME/clang ]; then
               COMMAND+=" PATH=$HOME/clang/bin:$PATH"
          fi
-         if [ ${{ inputs.aosp-gcc }} = true ]; then
+         if [ -d $HOME/gcc-64/bin ] || [ -d $HOME/gcc-32/bin ]; then
```

Comment:
?

Reply:
it's needed to set CROSS_COMPILE and CLANG_TRIPLE on third-party GCC. since aosp-gcc no longer enabled, so use gcc directory to make statement.